### ISSUE TITLE
[self-service]Customer Self-Service: What It Is & How to Do It Right

### ISSUE THREAD

**Issue by callmemingthx** [callmemingthx/4recording.github.io#4]
### https://blog.hubspot.com/service/customer-self-service

 It's not that I think the reps can't help or that they won't create a positive service experience, rather, it's that I'm a natural problem solver who likes to figure things out on my own. I prefer the challenge of coming up with my own solutions and won't reach out for customer support unless I truly believe I can't solve the problem by myself.

### what is self-service
Self-service simply means that customers must complete an action or task on their own without assistance from a company employee. 

### customer self-service
Customer self-service is proactive customer service that provides support for customers who want to find their own solutions. Rather than working with one of the company's customer service representatives, customers use self-service options to research and troubleshoot issues by themselves. Customer self-service is an important feature to include in your company's customer service offer as it can provide your customers with fast and readily available solutions.

### why you should provide customer self-service
1 Customer self-service is quickly becoming the preferred method for customer service. That's because self-service options provide customers with faster solutions that they can find on their own time. Rather than having to pick up a phone and wait on hold, customers can use your company's resources to look up answers to simple support questions.

2 Customer self-service is fairly easy to adopt as well as integrate into your customer service offer. If you don't offer any self-service options at the moment, one of the best places that any business can start is with its website.

#### 2 create product training for customers
What makes these courses great is the gamification aspect that comes along with taking them.
#### 3 provide flexible automation features
When it comes to customer-facing automation, it's important to make sure that these features are flexible and intuitive. Customers should be able to ask questions in different ways and receive consistent answers for every inquiry
#### 4 include escalation options
Forrester believes that in 2019 customers will start to demonstrate a backlash against chatbot support. This is because 60% of chatbots don't offer escalation options during the chat. Instead, customers have to call into the support line directly if they wish to escalate their issue to a live rep. This can ruin your customers' workflow, especially if they're just looking for a quick clarification on one or two details in the chatbot's response.
